src/components/Crop.tsx

```diff
@@ -134,11 +134,10 @@ const FourDivs = (
 );
 
 const cropStyle = (box: CropperBox, style: CSSProperties): CSSProperties => {
-  const { x, y, width, height } = box;
-
+  const { x, y, width, height, originX ,originY  } = box;
```

Comment:
Do run `yarn format` once.
You may want to set up your editor to auto-run prettier on save.

Reply:
sure